openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -56,29 +56,19 @@ def _do_third_party_auth(request):
             u"with backend_name {backend_name}".format(
                 username=username, backend_name=backend_name)
         )
-        message = _(
+        message = Text(_(
             u"You've successfully logged into your {provider_name} account, "
-            u"but this account isn't linked with an {platform_name} account yet."
-        ).format(
-            platform_name=platform_name,
-            provider_name=requested_provider.name,
-        )
-        message += "<br/><br/>"
-        message += _(
+            u"but this account isn't linked with an {platform_name} account yet. {br_tags}"
```

Comment:
You may want to either add a Translator note, or maybe change `{br_tags}` to be `{blank_line}` or something?